bin/MadGraph5_aMCatNLO/cards/examples/Z0Z0_LO/Z0Z0_LO_run_card.dat

```diff
@@ -0,0 +1,283 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+
+  .false.     = gridpack  !True = setting up the grid pack
+ -1.0	      = time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  average     = event_norm ! average/sum. Normalization of the weight in the LHEF
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  10000 = nevents ! Number of unweighted events requested 
```

Comment:
reduce this to 2000. pilot runs don't have to be big that much

Reply:
We kept it to 10000 events because we wanted to test the Gridpacks being generated from these cards.

Reply:
Not sure what you mean... this is just pilot run. Once you get the gridpacks you can get as many events as you want with by using runcmsgrid.sh in the gridpack.

Reply:
Okay, I am making the changes on the run cards now to have 2000 events.